src/main/java/cc/polyfrost/oneconfig/config/Config.java

```diff
@@ -273,6 +287,131 @@ protected final void generateOptionList(Object instance, Class<?> targetClass, O
         logger.trace("Finished generating option list for {} (targetting={})", mod.name, targetClass.getName());
     }
 
+    /**
+     * Adds a category to the option list, for internal use only
+     *
+     * @param instance    instance of target class
+     * @param targetClass which class to lookup into
+     * @param page        page to add options too
+     * @param mod         data about the mod
+     * @param category    category name
```

Comment:
these `@param` javadocs are mostly unnecessary and also the `category` field should be typed with our `Category` enum